src/react-queries/hooks/useQuoteEstimatedFeesQuery.ts

```diff
@@ -47,11 +67,16 @@ export const useQuoteEstimatedFeesQuery = ({
     const supportedEvmChainIds = getSupportedEvmChainIds()
     const cryptoAmount = depositAmountCryptoPrecision ?? repaymentAmountCryptoPrecision ?? '0'
     const assetId = repaymentAsset?.assetId ?? collateralAssetId
-    const quoteMemo = confirmedQuote?.quoteMemo ?? confirmedQuote?.quoteMemo ?? ''
-    const memo = supportedEvmChainIds.includes(fromAssetId(assetId).chainId as KnownChainIds)
-      ? utils.hexlify(utils.toUtf8Bytes(quoteMemo))
-      : quoteMemo
-    const to = confirmedQuote?.quoteInboundAddress ?? confirmedQuote?.quoteInboundAddress ?? ''
```

Comment:
and wat was that
![Added via Giphy](https://media0.giphy.com/media/3WmWdBzqveXaE/giphy.gif?cid=c9487743fo6jcxyf5nrart428dq9mxttnjsixjlpqesdioiz&ep=v1_gifs_search&rid=giphy.gif&ct=g)